packages/vue/src/components/molecules/SfProductCard/SfProductCard.html

```diff
@@ -1,4 +1,9 @@
 <div class="sf-product-card">
+  <button aria-label="Favorite this product" @click="toggleOnWishlist" class="sf-product-card__wishlist-icon">
```

Comment:
Maybe we can change aria-label for on wishlist product to "remove from wishlist" or something like that? :)